docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   spark-iceberg:
-    build: spark/
+    image: tabulario/spark-iceberg
```

Comment:
We should figure out a tagging strategy. Maybe tabulario/spark-iceberg:3.1.2_0.12.1?

I hate it, but it signifies almost all the important information.

We could also alias the most recent version as `latest` for convenience.

Reply:
Great idea. I just pushed without a tag which defaults to latest. In addition to different tags for spark+iceberg versions would we possibly want to actually version this test image as well? This might get hard to avoid a complex tag haha